### DISCUSSION TITLE
How are the example diagrams in the documentation drawn?

### DISCUSSION THREAD

**Discussion by aiktb** [vuejs/docs#2393] [General]
The images fit perfectly, and I'd like to use similar images elsewhere, but I can't find information about them.
Can someone please tell me, thanks.🥰
<img width="688" alt="slots dbdaf1e8" src="https://github.com/vuejs/docs/assets/104723527/ba505400-3b15-472f-8887-3fc9e176c6f8">
<img width="688" alt="prop-drilling 11201220" src="https://github.com/vuejs/docs/assets/104723527/8a27e54e-b003-4720-8981-2ccdc8412f06">

**Comment by bencodezen** [comment]
Great question @aiktb! 

Believe it or not, they were drawn in Figma. No components libraries or anything 😅

🎩 Credit for these diagrams go to @yyx990803!

If you have any other questions, don't hesitate to reopen this discussion!
